### DISCUSSION TITLE
`govet` vs `go vet`

### DISCUSSION THREAD

**Discussion by silverwind** [golangci/golangci-lint#4363] [Q&A] [answered]
Are the [`govet`](https://golangci-lint.run/usage/linters/#govet) linter and [`go vet`](https://pkg.go.dev/cmd/vet) the same thing? I'm looking to use `go vet` and am wondering whether the `govet` linter would cover everything that `go vet` does. The docs are not clear about this at all.

**Comment by ldez** [comment, marked as answer]
yes and no.

`go vet` is a command line that runs [passes](https://github.com/golang/tools/tree/master/go/analysis/passes).

`govet` is an internal golangci-lint linter that runs the [passes](https://github.com/golang/tools/tree/master/go/analysis/passes) used by `go vet`.

`govet` does the same analysis as `go vet`.

**Reply by silverwind** [reply]
Thanks, I opened https://github.com/golangci/golangci-lint/pull/4366 to clarify the docs.

**Comment by Antonboom** [comment]
@silverwind be careful with fact that in the same time `go vet` could doing more checks than `golangci-lint/govet` (until we update the last one).
